pkg/morph/client/netmap/new_epoch.go

```diff
@@ -6,27 +6,12 @@ import (
 	"github.com/nspcc-dev/neofs-node/pkg/morph/client"
 )
 
-// NewEpochArgs groups the arguments
-// of new epoch invocation call.
-type NewEpochArgs struct {
-	number int64 // new epoch number
-
-	client.InvokePrmOptional
-}
-
-// SetEpochNumber sets the new epoch number.
-func (a *NewEpochArgs) SetEpochNumber(v int64) {
-	a.number = v
-}
-
-// NewEpoch invokes the call of new epoch method
-// of NeoFS Netmap contract.
-func (c *Client) NewEpoch(args NewEpochArgs) error {
+// NewEpoch updates NeoFS epoch number through
+// Netmap contract call.
+func (c *Client) NewEpoch(epoch uint64) error {
 	prm := client.InvokePrm{}
-
 	prm.SetMethod(newEpochMethod)
-	prm.SetArgs(args.number)
-	prm.InvokePrmOptional = args.InvokePrmOptional
+	prm.SetArgs(epoch)
```

Comment:
You should either convert it into `int64` or update code in `pkg/morph/client/client.go#384`. 

```diff
	case int64: // TODO: add other numerical types
		result.Type = sc.IntegerType
+	case uint64:
+		result.Type = sc.IntegerType
+		result.Value = int64(v)
```

I didn't check all the code, maybe there is more cases when we pass `uint64` into args in this PR, so maybe change above is more preferred.

Reply:
The problem is that this is `smartcontract.Parameter` which has a specific logic when marshaling in JSON https://github.com/nspcc-dev/neo-go/blob/master/pkg/smartcontract/parameter.go#L67 . Similar thing happens during unmarshaling on the server side https://github.com/nspcc-dev/neo-go/blob/01d15ff473f9984023e9c1e04be107981acec80e/pkg/rpc/request/txBuilder.go#L65 .

Reply:
nspcc-dev/neo-go#2350